packages/rule-tester/tests/eslint-base/eslint-base.test.js

```diff
@@ -473,7 +473,7 @@ describe("RuleTester", () => {
                     "bar = baz;"
                 ],
                 invalid: [
-                    { code: "var foo = bar; var baz = quux", errors: [{ type: "VariableDeclaration" }, null] }
+                  { code: "var foo = bar; var baz = quux", errors: [{ message: "Bad var.", type: "VariableDeclaration" }, null] }
```

Comment:
```suggestion
                    { code: "var foo = bar; var baz = quux", errors: [{ message: "Bad var.", type: "VariableDeclaration" }, null] }
```

We should try to keep this file 1:1 with the upstream tests, even in formatting!

Reply:
Done